src/InputText/InputText.test.jsx

```diff
@@ -62,5 +62,10 @@ describe('<InputText />', () => {
       expect(input.hasClass('first')).toEqual(true);
       expect(input.hasClass('last')).toEqual(true);
     });
+
+    it('should render with the readOnly property if set', () => {
+      const wrapper = mount(<InputText {...props} readOnly />);
+      expect(wrapper.props().readOnly).toEqual(true);
+    });
```

Comment:
It would be great to add a test to check that it defaults to not having a readOnly prop set too.